eng/Versions.props

```diff
@@ -247,6 +247,7 @@
     <IdentityServer4StoragePackageVersion>3.0.0</IdentityServer4StoragePackageVersion>
     <IdentityServer4EntityFrameworkStoragePackageVersion>3.0.0</IdentityServer4EntityFrameworkStoragePackageVersion>
     <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
+    <MicrosoftAspNetCoreAuthPackageVersion>5.0.0-preview.8.20404.9</MicrosoftAspNetCoreAuthPackageVersion>
```

Comment:
For consistency with other `$(*PackageVersion)` variables, please spell out the two package names completely. (We also don't abbreviate "auth" in names.)